src/transformers/models/vision_encoder_decoder/modeling_vision_encoder_decoder.py

```diff
@@ -336,8 +336,20 @@ def from_pretrained(cls, pretrained_model_name_or_path, *model_args, **kwargs):
                 del tf_model
                 gc.collect()
 
+                attn_implementation = kwargs.get("attn_implementation", None)
+                kwargs_encoder_decoder = {}
+                if attn_implementation:
+                    kwargs_encoder_decoder = {
+                        "encoder_attn_implementation": attn_implementation,
+                        "decoder_attn_implementation": attn_implementation,
+                    }
+
                 model = VisionEncoderDecoderModel.from_encoder_decoder_pretrained(
-                    encoder_dir, decoder_dir, encoder_from_tf=True, decoder_from_tf=True
+                    encoder_dir,
+                    decoder_dir,
+                    encoder_from_tf=True,
+                    decoder_from_tf=True,
+                    **kwargs_encoder_decoder,
```

Comment:
I don't think it is needed in that case. When you use `from_encoder_decoder_pretrained` you need this but l.371 uses `from_pretrained` and it will use the our keyword argument[ as intended](https://github.com/huggingface/transformers/blob/ae945443cc2cc1045c03ab266ddb2767e8dcc221/src/transformers/models/vision_encoder_decoder/modeling_vision_encoder_decoder.py#L192) during class instantiation.